server/src/agents/executor/toolkit.ts

```diff
@@ -296,17 +365,147 @@ export const getExecutorToolkit = (account: Account): Record<string, Tool> => {
 
           return {
             success: true,
-            result: `Transaction executed successfully. Transaction hashes: ${hashes.join(", ")}`
+            result: `Transaction executed successfully. Transaction hashes: ${hashes.join(", ")}`,
+          };
+        } catch (error) {
+          return {
+            success: false,
+            result: null,
+            error: error instanceof Error ? error.message : "Unknown error",
+          };
+        }
+      },
+    },
+    crossChainNativeTokenTranfer: {
+      description: `
+      Cross Chain native Token transfer from source chain to destination chain using wormhole SDK. Remember to ask the user for source chain, destination chain and amount.
```

Comment:
I think for autonomous actions we are expecting the user to provide us with source , dstn and amount